client/dashboard/sites/staging-site-delete-modal/index.tsx

```diff
@@ -21,29 +24,56 @@ export default function StagingSiteDeleteModal( {
 	onClose: () => void;
 } ) {
 	const router = useRouter();
+	const queryClient = useQueryClient();
 	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
 
 	const productionSiteId = site.options?.wpcom_production_blog_id;
-	const mutation = useMutation( stagingSiteDeleteMutation( site.ID, productionSiteId ?? 0 ) );
+
+	const deleteMutation = useMutation( {
+		...stagingSiteDeleteMutation( site.ID, productionSiteId ?? 0 ),
+		onSuccess: () => {
+			setTimeout( () => {
+				queryClient.invalidateQueries( {
+					queryKey: [ 'automated-transfer-status', site.ID ],
+				} );
```

Comment:
This is the preferred approach :)

```suggestion
				queryClient.invalidateQueries( automatedTransferStatusQuery( site.ID ) );
```

Reply:
Switched in https://github.com/Automattic/wp-calypso/pull/104557/commits/a50441aca27d3b498f0a15bfe70606877960696a